### DISCUSSION TITLE
Does anyone agree with this about how it is fun to use assembly until you end up having to spend days on the same low-level issue(s)?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#141093] [Programming Help]
### Body

Does anyone agree with this about how it is fun to use assembly until you end up having to spend days on the same low-level issue(s)? 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by MikaStiebitz** [comment]
Haha, agree :D
